lib/portage/_sets/dbapi.py

```diff
@@ -168,14 +168,32 @@ def _filter(self, atom):
             return False
         (values,) = self._metadatadb.aux_get(ebuild, [self._variable])
         values = values.split()
+
+        if "DEPEND" in self._variable:
+            include_atoms = []
+            for include in self._includes:
+                include_atoms.append(Atom(include))
+
+            for x in use_reduce(values, token_class=Atom):
+                if not isinstance(x, Atom):
+                    continue
+
+                for include_atom in include_atoms:
+                    print(f" Checking if {x=} intersects with {include_atom=}")
```

Comment:
Support for `{x=}` was added in [Python 3.8](https://docs.python.org/3/whatsnew/3.8.html#f-strings-support-for-self-documenting-expressions-and-debugging), but we claim to support Python 3.7.

Reply:
Oh, I suppose this is just a leftover debug statement anyway.